### ISSUE TITLE
Move prepare script to the root folder?

### ISSUE THREAD

**Issue by olayway** [flowershow/flowershow#205]
```json
    "prepare": "cd ../.. && husky install templates/default/.husky",
```

The `prepare` script in our default template's `package.json` will fail to execute when running `npm install` not within `flowershow` repository (e.g. as part of `npx flowershow install`).

### Solution A (wrong)
Running `npm install` with `--ignore-scripts` flag.
```js
// packages/cli/lib/Installer.js
// ...
      const { stdout, stderr } = await execa('npm', [ 'install', '--ignore-scripts' ], { cwd: flowershowDir });
// ...
```
This will work for `npm install` but will break the build.

### Solution B (current)
Removing `prepare` script before running `npm install` in the CLI.

```js
// packages/cli/lib/Installer.js
// ...
      await execa('npm', [ 'set-script', 'prepare', '' ], { cwd: flowershowDir });
      const { stdout, stderr } = await execa('npm', [ 'install'], { cwd: flowershowDir });

// ...
```

This will change the `prepare` script in `package.json` to:
```json
    "prepare": "",
```

**Comment by rufuspollock** [comment]
@olayway reading this i'm not clear what work is involved? Do we need to move the prepare script up still.  Could you also give an estimate of time for this so i can prioritize better 😄

**Comment by olayway** [comment]
Closed with #241
